<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectPeakDialog</class>
 <widget class="QDialog" name="SelectPeakDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select peak</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="ql_selectPeak">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select peak:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="qtbv_listOfPeaks">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="qpb_select">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Use selected peaks</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="qpb_selectAll">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Use all peaks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
